### PR TITLE
Making quickblock optional

### PR DESCRIPTION
**Pull request by josephwhite**
A simple edit to make quickblock an optional feature.

### CODE REVIEW

File: twitter.gt2eb.user.js [hunk 1]
```diff
@@ -372,6 +372,7 @@
     hideWhoToFollow:          false,
     hideTranslateTweetButton: false,
     hideMessageBox:           true,
+    enableQuickBlock:         true,
```
Comment (Bl4Cc4t) [inline twitter.gt2eb.user.js:375]:
```suggestion
    enableQuickBlock:         false,
```
On second thought, diabling it by default is even better.

**Review by Bl4Cc4t — CHANGES_REQUESTED** [review]
You're the first to actually mess with the code, that's so cool to see!
I should've made it optional in the first place I guess.
Only one change from my side!

File: twitter.gt2eb.user.js (hunk 1 above)
Comment (josephwhite) [reply twitter.gt2eb.user.js]: Thank you for the suggestion and I'm glad to help! If there is also a better place in the settings menu to put the toggle, let me know.

Comment (Bl4Cc4t) [reply twitter.gt2eb.user.js]: I think the "Other" section is fine

**Pull request merged**